tests/run/builtin_divmod.py

```diff
@@ -0,0 +1,79 @@
+# mode: run
+
+import cython
+
+def divmod_regular(a,b):
+    """
+    >>> divmod_regular(10,5)
+    (2, 0)
+    >>> divmod_regular(9191,4096)
+    (2, 999)
+    >>> divmod_regular(10000,10010)
+    (0, 10000)
+    >>> divmod_regular(-999999,-111111)
+    (9, 0)
+    >>> divmod_regular(-888888,-11111)
+    (80, -8)
+    >>> divmod_regular(-10000,-10086)
+    (0, -10000)
+    >>> divmod_regular(5,-1)
+    (-5, 0)
+    >>> divmod_regular(-40,3)
+    (-14, 2)
+    >>> divmod_regular(11,-3)
+    (-4, -1)
+    >>> divmod_regular(0,9)
+    (0, 0)
+    >>> divmod_regular(0,-987654321)
+    (0, 0)
+    """
+    return divmod(a,b)
+
+
+def divmod_by_0(a,b):
+    """
+    >>> divmod_by_0(33,0) #doctest: +ELLIPSIS
+    Traceback (most recent call last):
+    ZeroDivisionError: ...
+    """
+    return divmod(a,0)
```

Comment:
Let's keep it invisible to the compiler that we're dividing by 0 here.
```suggestion
def divmod_by_0(a,b):
    """
    >>> divmod_by_0(33,0) #doctest: +ELLIPSIS
    Traceback (most recent call last):
    ZeroDivisionError: ...
    """
    return divmod(a,b)
```